src/components/app/data/services/enterpriseCustomerUser.js

```diff
@@ -204,3 +204,16 @@ export async function updateUserCsodParams({ data }) {
   const url = `${getConfig().LMS_BASE_URL}/integrated_channels/api/v1/cornerstone/save-learner-information`;
   return getAuthenticatedHttpClient().post(url, data);
 }
+
+/**
+ * Helper function to uunlink enterprise customer user by making a POST API request.
+ * @param {Object} params - The parameters object.
+ * @param {Object} params.enterpriseCustomerUser - The enterprise customer user that should be unlinked.
+ * @returns {Promise} - A promise that resolves when the active enterprise customer user is unlinked.
+ */
+export async function postUnlinkUserFromEnterprise(enterpriseCustomerUserUUID, userEmail) {
```

Comment:
perhaps a better name would be `unlinkUserFromEnterprise`?

Reply:
I added `post` here to mention that it's a post request for better readability.